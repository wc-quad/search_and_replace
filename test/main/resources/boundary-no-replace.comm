R flea shoe
R fly die
R flee shoo
Q
