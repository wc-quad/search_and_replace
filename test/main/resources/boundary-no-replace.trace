Give a sequence of commands of the form:
s word             [search]
r word replacement [replace]
Type 'q' to quit
< A flea and a fly in a flue
> A shoe and a fly in a flue
< "Let us fly!" said the flea.
> "Let us fly!" said the shoe.
< A shoe and a fly in a flue
> A shoe and a die in a flue
< Said the fly, "let us flee!"
> Said the die, "let us flee!"
< "Let us fly!" said the shoe.
> "Let us die!" said the shoe.
< Said the die, "let us flee!"
> Said the die, "let us shoo!"
