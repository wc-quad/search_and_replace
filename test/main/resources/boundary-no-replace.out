A shoe and a die in a flue
Were imprisoned, so what could they do?
Said the die, "let us shoo!"
"Let us die!" said the shoe.
So they flew through a flaw in the flue.
